content/docs/examples/kubeflow-samples.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Kubeflow Samples"
+description = "Examples that demonstrate machine learning with Kubeflow"
+weight = 10
++++
+
+{{% blocks/content-item %}}
```

Comment:
Please remove the {{% blocks/content-item %}} and {{% /blocks/content-item %}} shortcodes for this sentence.

Reply:
Good catch, I removed it